generators/client/templates/angularjs/src/main/webapp/app/services/auth/_account.service.js

```diff
@@ -26,7 +26,7 @@
     Account.$inject = ['$resource'];
 
     function Account ($resource) {
-        var service = $resource('<%- apiUaaUrlPrefix %>api/account', {}, {
+        var service = $resource('<%- apiUaaPath %>api/account', {}, {
```

Comment:
@ruddell just for my understanding as my memory is quite weak, why is the server api url not needed for these?

Reply:
There is no SERVER_API_URL variable in AngularJS

Reply:
Sorry didn't notice it was ng1 file